docs/connectors/rhoas-cli-getting-started-connectors/README.adoc

```diff
@@ -387,13 +394,23 @@ To check the status:
 $ rhoas connector list
 ----
 
-. Verify that the your sink Connectors instance is receiving messages by viewing your link:https://webhook.site[Webhook.site^] page in a web browser.
+. Verify that your sink {connectors} instance is receiving messages by viewing your https://webhook.site[Webhook.site^] page in a web browser.
+
+[id="proc-commands-managing-connectors_{context}"]
+== Commands for managing {connectors} instances
+
+[role="_abstract"]
+For more information about the `rhoas connector` commands that you can use to manage your {connectors} instances, use the following command help:
 
+* `rhoas connector namespace -h` for managing {connectors} namespaces
+* `rhoas connector type -h` for viewing the available types of connectors
+* `rhoas connector list -h`` for listing {connectors} instances
+* `rhoas connector build -h` for building configuration files
+* `rhoas connector create -h` for creating {connectors} instances
 
 [role="_additional-resources"]
 .Additional resources
-* To access the `rhoas connector` help page, type `rhoas connector -h`
-{base-url-cli}{command-ref-url-cli}[_CLI command reference (rhoas)_^]
+* {base-url-cli}{command-ref-url-cli}[_CLI command reference (rhoas)_^]
```

Comment:
You can remove the bullet formatting again, now that you've added the new Commands section :)